modin/pandas/base.py

```diff
@@ -2021,37 +1903,22 @@ def _stat_operation(
         """
         axis = self._get_axis_number(axis)
         validate_bool_kwarg(skipna, "skipna", none_allowed=False)
-        if level is not None:
-            return self._default_to_pandas(
-                op_name,
-                axis=axis,
-                skipna=skipna,
-                level=level,
-                numeric_only=numeric_only,
-                **kwargs,
-            )
-        # If `numeric_only` is None, then we can do this precheck to whether or not
-        # frame contains non-numeric columns, if it doesn't, then we can pass to a query compiler
-        # `numeric_only=False` parameter and make its work easier in that case, rather than
-        # performing under complicate `numeric_only=None` parameter
+        if op_name == "median":
+            numpy_compat.function.validate_median((), kwargs)
+        elif op_name in ("sem", "var", "std"):
+            val_kwargs = {k: v for k, v in kwargs.items() if k != "ddof"}
+            numpy_compat.function.validate_stat_ddof_func((), val_kwargs, fname=op_name)
+        else:
+            numpy_compat.function.validate_stat_func((), kwargs, fname=op_name)
+
         if not numeric_only:
-            try:
-                self._validate_dtypes(numeric_only=True)
-            except TypeError:
-                if numeric_only is not None:
-                    raise
-            else:
-                numeric_only = False
+            # fix for 'test_reduce_specific'
```

Comment:
redundant, should be removed before merging. look for other similar places

Reply:
removed